### ISSUE TITLE
"wrapSelection.patterns" won't work in settings.json

### ISSUE THREAD

**Issue by sepehrooo** [gko/wrap#5]
I tired adding your wrapSelection.patterns from your readme file to my settings.json: 
```
"wrapSelection.patterns": {
		"log": "console.log(`${text}`, ${text})",
		"promise": "new Promise((yeah, nah) => yeah(${text}))",
		"=>": "() => ${text}"
	}
```
But VS code says "Unknown configuration setting".
And I can't add custom wrapping patterns.

Am I doing anything wrong?

**Comment by zf-1** [comment]
same question

**Comment by eksortso** [comment]
Having the same problem here. And when I search Settings for `@ext:konstantin.wrapSelection`, nothing comes up.

**Comment by neelansh15** [comment]
I would like to know this too, as I need to wrap stuff in <div> elements quite often, and I really need a custom pattern.

**Comment by RyanRoberts** [comment]
Yep. Not working 😆

**Comment by gko** [comment]
Fixed in version 0.7.0
Sorry for the delay 😔
